src/pip/_internal/locations.py

```diff
@@ -146,12 +158,12 @@ def distutils_scheme(dist_name, user=False, home=None, root=None,
         extra_dist_args = {"script_args": ["--no-user-cfg"]}
     else:
         extra_dist_args = {}
-    dist_args = {'name': dist_name}
+    dist_args = {'name': dist_name}  # type: Dict[str, Union[str, List[str]]]
     dist_args.update(extra_dist_args)
 
     d = Distribution(dist_args)
-    d.parse_config_files()
-    i = d.get_command_obj('install', create=True)
+    d.parse_config_files()  # type: ignore
```

Comment:
A comment mentioning why these were ignored, just above the code would be helpful.